checker/src/main/java/org/checkerframework/checker/determinism/DeterminismAnnotatedTypeFactory.java

```diff
@@ -847,4 +847,13 @@ public boolean isSubtype(AnnotationMirror subAnno, AnnotationMirror superAnno) {
             return super.isSubtype(subAnno, superAnno);
         }
     }
+
```

Comment:
Since the rest of this is implement in the framework.  Please move this up to GenericAnnotatedTypeFactory.